src/perception/tag_detector.cpp

```diff
@@ -70,13 +70,13 @@ void FiducialsNode::configCallback(mrover::DetectorParamsConfig &config, uint32_
     mDetectorParams->polygonalApproxAccuracyRate = config.polygonalApproxAccuracyRate;
 }
 
-void FiducialsNode::ignoreCallback(std_msgs::String const &msg) {
+void FiducialsNode::ignoreCallback(std_msgs::String const& msg) {
```

Comment:
I'm struggling to find anything about this online. Is `std_msgs::String const& msg` here the same as `const std_msgs::String& msg`?

Reply:
K nvm looks like that is in fact the same

Reply:
Yes it is